### ISSUE TITLE
iOS App - Push Notifications Not Working

### ISSUE THREAD

**Issue by laurin1** [gitterHQ/gitter#1719]
iOS 10.3.2, iPhone 7+ 

Push notifications have never worked. I've checked all notification settings. I deleted the app and reinstalled. They never work, not even intermittently.

**Comment by mydigitalself** [comment]
@laurin1 when did you sign up? (just trying to get a timeframe of "never")

Did you know that push notifications won't be delivered if you are online on Gitter - i.e. if you have an open Gitter tab or if you are connected to Gitter via IRC.

**Comment by laurin1** [comment]
I don't know for sure, but I've had the app installed for a long time (a year maybe?). I didn't use the app much after initial install, so I didn't bother to ever attempt to troubleshoot the problem. I started using it again a few weeks ago.

**Comment by laurin1** [comment]
> Did you know that push notifications won't be delivered if you are online on Gitter - i.e. if you have an open Gitter tab or if you are connected to Gitter via IRC.

Yes, I've read about that - I'm not sure what that means. Does that mean if Gitter is running on a machine, but I'm not using it, I won't get notifications? I leave Gitter running all the time, but most of the time it's int the System Tray (Windows 10 Pro x64).
